src/Components/GameMainPage/Canvas/Canvas.jsx

```diff
@@ -20,9 +20,10 @@ const Canvas = ({thingy}) => {
     setUserWithId(x);
   }, [currentUser]);
 
-
   const setup = (p5, canvasParentRef) => {
-    const canva = p5.createCanvas(thingy.offsetWidth, thingy.offsetHeight - 100).parent(canvasParentRef);
+    const canva = p5
+      .createCanvas(thingy.offsetWidth, thingy.offsetHeight - 100)
```

Comment:
Is `thingy` a p5 thing?  If not a more explicit name might help it be easier to grok